app/decorators/case/base_decorator.rb

```diff
@@ -67,6 +67,12 @@ def trigger_case_marker
           "Trigger" +
           h.content_tag(:span, " case", class: "visually-hidden")
       end
+    elsif "#{object.type_abbreviation.downcase}" == "offender_sar_complaint" && high_priority?
```

Comment:
There are decorators per case type, so it would be better to overload this method in the  offender complaint decorator.